### PR TITLE
fix: AU-1473: Fix headers in grey box on page 1 of applications

### PR DESCRIPTION
**Pull request by strangeworlder**
# [AU-1473](https://helsinkisolutionoffice.atlassian.net/browse/AU-1473)

## What was done

* On page 1 of forms there was a H2 with the title "Hakijan tiedot" that had a H3 inside it with the title "Hakijan tiedot" with a fieldset inside it with the legend "Hakijan tiedot". These were changed to different names
* The H3 inside the grey box was 33% wide, it is now of proper size.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-1473-grey-box`
  * `make fresh`
* Run `make drush-cr`

## How to test

* [ ] See that the headline in the grey box, if it was "hakijan tiedot" is now "haetut tiedot"
* [ ] Check that code follows our standards

## Designers review

* [ ] This PR does not need designers review
* [ ] This PR has been visually reviewed by a designer (Name of the designer)

## Other PRs

* Link to other PR

[AU-1473]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: conf/cmi/language/en/webform.webform.kasko_ip_lisa.yml
Comment (suvikankare) [inline conf/cmi/language/en/webform.webform.kasko_ip_lisa.yml:1]: Oisko nää kuitenkin retrieved?

**Review by suvikankare — CHANGES_REQUESTED** [review]

File: conf/cmi/language/en/webform.webform.kasko_ip_lisa.yml
Comment (strangeworlder) [reply conf/cmi/language/en/webform.webform.kasko_ip_lisa.yml:1]: Vois olla :D

Comment (strangeworlder) [reply conf/cmi/language/en/webform.webform.kasko_ip_lisa.yml:1]: Fix'd

**Comment by strangeworlder** [comment]
typo fiksattu @suvikankare

**Review by suvikankare — APPROVED** [review]

**Pull request merged**